test/integration/annotation/modification_functions.go

```diff
@@ -143,3 +144,34 @@ func addAnnotation(t *testing.T, testRunner base.ClusterTestRunner) {
 		assert.Assert(t, err)
 	}
 }
+
+// debugAnnotatedResources prints current status for the exposed resources
+func debugAnnotatedResources(t *testing.T, testRunner base.ClusterTestRunner) {
+	pub, _ := testRunner.GetPublicContext(1)
+	prv, _ := testRunner.GetPrivateContext(1)
+
+	log.Printf("Debugging exposed resources (by annotation):")
+	i := 0
+	for _, cluster := range []*client.VanClient{pub.VanClient, prv.VanClient} {
+		// Retrieving the deployment
+		dep, _ := cluster.KubeClient.AppsV1().Deployments(cluster.Namespace).Get("nginx", v1.GetOptions{})
+		log.Printf("Deployment: %s - Annotations: %s", dep.Name, dep.Annotations)
+		if len(dep.Annotations) > 0 {
+			i++
+		}
+
+		// Retrieving services
+		svcList, _ := cluster.KubeClient.CoreV1().Services(cluster.Namespace).List(v1.ListOptions{
+			LabelSelector: "app=nginx",
+		})
+
+		for _, svc := range svcList.Items {
+			log.Printf("Service   : %s - Annotations: %s", svc.Name, svc.Annotations)
+			if _, ok := svc.Annotations[types.ProxyQualifier]; len(svc.Annotations) > 0 && ok {
```

Comment:
is len check necessary?

Reply:
Nope, it was there before I verified the proxy qualifier specifically. I removed it now.